hwy/ops/x86_256-inl.h

```diff
@@ -4207,6 +4207,47 @@ HWY_API Vec256<double> ConvertTo(Full256<double> dd, const Vec256<int64_t> v) {
 #endif
 }
 
+HWY_API Vec256<float> ConvertTo(HWY_MAYBE_UNUSED Full256<float> dd,
+                                const Vec256<uint32_t> v) {
+#if HWY_TARGET <= HWY_AVX3
+  return Vec256<float>{_mm256_cvtepu32_ps(v.raw)};
+#else
+  // Based on wim's approach (https://stackoverflow.com/questions/34066228/)
+  const Repartition<uint32_t, decltype(dd)> du32;
+  const Repartition<int32_t, decltype(dd)> d32;
+
+  const auto msk_lo = Set(du32, 0xFFFF);
+  const auto cnst2_16_flt = Set(dd, 65536.0f); /* 2^16 */
+  const auto v_lo = BitCast(d32, And(v, msk_lo)); /* Extract the 16 lowest significant bits of v and cast to signed int */
+  const auto v_hi = BitCast(d32, ShiftRight<16>(v));
+  return MulAdd(cnst2_16_flt, ConvertTo(dd, v_hi), ConvertTo(dd, v_lo));
+#endif
+}
+
+HWY_API Vec256<double> ConvertTo(HWY_MAYBE_UNUSED Full256<double> dd,
+                                  const Vec256<uint64_t> v) {
+#if HWY_TARGET <= HWY_AVX3
```

Comment:
Should this be `>=`, or is what the comparison does unintuitive?
I.e., without AVX3 (e.g. only AVX2) you'd need to run the fallback. Only AVX3(+) provides `_mm256_cvtepu64_pd`.

Reply:
Yes, but the highest-priority target has the smallest numerical value :-)

Reply:
I was also confused about it at first.

Reply:
That's right :) Thanks for raising this point, I will document it and explain why `<= HWY_AVX3` works and is desirable.